lighthouse-core/audits/manifest-display.js

```diff
@@ -37,7 +37,8 @@ class ManifestDisplay extends Audit {
    * @return {boolean}
    */
   static hasRecommendedValue(val) {
-    return (val === 'fullscreen' || val === 'standalone' || val === 'browser');
+    return (val === 'fullscreen' || val === 'standalone' ||
```

Comment:
This could be: `['browser', 'fullscreen', 'minimal-ui',  'standalone'].indexOf(val) !== -1;`